x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -62,15 +62,6 @@ export function renderAsRedirectTo(to: string) {
 // If you provide an inline function to the component prop, you would create a
 // new component every render. This results in the existing component unmounting
 // and the new component mounting instead of just updating the existing component.
-//
-// This means you should use `render` if you're providing an inline function.
-// However, the `ApmRoute` component from @elastic/apm-rum-react, only supports
-// `component`, and will give you a large console warning if you use `render`.
-//
-// This warning cannot be turned off
-// (see https://github.com/elastic/apm-agent-rum-js/issues/881) so while this is
-// slightly more code, it provides better performance without causing console
-// warnings to appear.
```

Comment:
It's possible to not inline the redirects and continue to use `component` for the RUM agent. It would be a lot of boilerplate, but we could make it work:
```
const RedirectToServices = renderAsRedirectTo('/services');
...
{
  component: RedirectToServices,
}
```

Reply:
We don't need to though. The problem with using `render` is the mounting/unmounting, which doesn't matter as much when we're just doing a redirect.